user_docs/en/userGuide.t2t

```diff
@@ -267,6 +267,64 @@ To open NVDA's menu, press ``NVDA+n``.
 To open NVDA's general settings dialog directly, press ``NVDA+control+g``.
 Many settings screens have keystrokes to open them directly, such as ``NVDA+control+s`` for synthesizer, or ``NVDA+control+v`` for other voice options.
 
+++ Add-ons ++[Addons]
+Add-ons are programs which provide new or changed functionality for NVDA.
+Add-ons are developed by the NVDA community, or external companies.
+As with any software, it is important to be confident in the developer of an add-on before using it.

```

Comment:
```suggestion
As with any software, it is important to be confident in the developer of an add-on before using it.
You can do this by researching their reputation (eg. how long have they been contributing, look for user feedback in the NVDA user or add-on groups), verifying that the add-on is regularly updated, and seeking community feedback through the forums or social media.
```

Reply:
We should probably mention the "Community reviews" here?

I'm also not sure we should add too much more here when we don't actually mention this in 13.2 - perhaps we should expand on this in 13.2 and add a brief mention here that "As with any software, it is important to be confident in the developer of an add-on before using it.  Section 13.2.1 contains tips on researching add-ons and authors, such as community reviews"?

Reply:
That sounds like a better solution.

Reply:
I have created PR #16510 to add @gerald-hartig's suggestions to the main add-on section.  I'd like to propose that PR be accepted at the same time or before this one, so the following makes sense:
```suggestion
As with any software, it is important to be confident in the developer of an add-on before using it.
Please refer to [Installing Add-ons #AddonStoreInstalling] for ways to verify add-ons prior to installation.
```